docs/dev/cli/sauce-connect-proxy.md

```diff
@@ -149,21 +150,36 @@ __Shorthand__: `-t`
 <p><small>| OPTIONAL | STRING |</small></p>
 
 :::caution
-Effective with Sauce Connect Proxy version 4.7.0, we recommend using [`--region`](/dev/cli/sauce-connect-proxy/#--region) instead. `--rest-url` will eventually be deprecated. Download the latest SC version [here](/secure-connections/sauce-connect/installation/).
+As this option will soon be deprecated, we recommend using the [`--region`](/dev/cli/sauce-connect-proxy/#--region) flag, compatible with Sauce Connect versions 4.7.0 and above.
 :::
 
-__Description__: Sets your [Sauce Labs regional data center REST API URL](#data-center-endpoints) (e.g., EU-Central, US-West).<br/>
+__Description__: Sets the [Sauce Labs data center endpoint URL](/basics/data-center-endpoints) of the location where the device you're testing on is hosted.<br/>
 __Enviroment variable__: `SAUCE_REST_URL`<br/>
 __Shorthand__: `-x`
 
+```bash
```

Comment:
The endpoint to connect to **EU-Central** is listed twice (https://eu-central-1.saucelabs.com/rest/v1). Should it be replaced with the endpoint for **US-East** (https://us-east-1.saucelabs.com/rest/v1) + should we add the endpoint for **US-West** (https://api.us-west-1.saucelabs.com/rest/v1) as well?

Reply:
good catch! I will message you separately about this.